css/app.css

```diff
@@ -27,8 +27,9 @@
     --title-color: #2196F3;
     --progress-color: #1976D2;
 
-    --selected-color: #3b99fc;
-    --selected-color2: rgba(59, 153, 252, 0.1);
+    --selected-color: orangered;
```

Comment:
probably change the colors to go with the current color scheme. These colors look too flashy in the video